requirements/test.txt

```diff
@@ -1,6 +1,10 @@
 -r base.txt
 django>=1.11,<2.0
+celery>=3.1.25,<4.0
 -e git://github.com/edx/xblock-sdk.git@master#egg=xblock-sdk==master
+-e git+https://github.com/edx/xblock-utils.git@v1.0.5#egg=xblock-utils==v1.0.5
```

Comment:
Any particular reason for not using 1.2.2 here, and for installing from GitHub as opposed to [PyPI](https://pypi.org/project/xblock-utils/)?

Reply:
The only reason was that it didn't use to be on PyPI, and I never checked to see if they pushed it up.  Fixed on latest push.